app.js

```diff
@@ -78,6 +96,28 @@ async function runCommand(pluginParameters) {
   return stdout;
 }
 
+function extractUserFromJWT(token) {
```

Comment:
I think we can move this function and `validateCertificate` to `helm-cli.js` file. They fit better in `helm-cli` namespace rather than `app.js`, or maybe create a separate file `helpers.js` and move the `logToActivityLog` function there too.

Reply:
`validateCertificate` has nothing to do with Helm nor any CLI.

I agree with `helpers.js`. I haven't separated it initially, because the code is tiny, but in the end I agree to keep the proper layers separation.

Reply:
Fixed